src/lib/fs.js

```diff
@@ -23,4 +23,13 @@ const fileExistsAsync = async filePath => {
   }
 }
 
-module.exports = { statAsync, readFileAsync, readFileAsyncCatchError, writeFileAsync, fileExistsAsync }
+const isFileAsync = async filePath => {
```

Comment:
Should we use `path-type`?
https://github.com/sindresorhus/path-type#isfilepath

Reply:
Yes, this is much better!

Reply:
Done in https://github.com/netlify/cli/pull/1305/commits/df18aca5268c47dc175cf2361fca54b71c7b7892. Still had to wrap it with `try/catch` since `path-type` throws:
https://github.com/sindresorhus/path-type/blob/8e4e0a24531d639b5b9f1eb8b08c01e9336578e1/index.js#L18

Reply:
This would only throw if there is something wrong with the filesystem, such as permission error. Shouldn't we propagate those errors?

Reply:
Sure, let me handle that

Reply:
Done in https://github.com/netlify/cli/pull/1305/commits/6e2f29cd7b7ed2fe7e69643b00542ac99fbd78f6